pilot/pkg/bootstrap/server.go

```diff
@@ -113,8 +113,7 @@ var (
 		plugin.Snidnat,
 	}
 
-	// Global mutex
-	globalMutex = sync.Mutex{}
+	runOnce = sync.Once{}
```

Comment:
It is not clear what this runOnce does. A better name could be enableHistogramOnce